content/blog/automation-api-as-platform/index.md

```diff
@@ -0,0 +1,275 @@
+---
+title: "Build Self-Service Cloud Infrastructure with Automation API "
+date: 2020-11-09
+meta_desc: "With Pulumi's Automation API, you can build a REST API for creating, reading, updating, and deleting (CRUD) cloud resources."
+meta_image: automation_api.png
+authors:
+    - sophia-parafina
+tags:
+    - Automation API
+    - CRUD
+---
+
+If you could create infrastructure without using a cloud provider's console, a CLI, or a templating engine, what would you build? Pulumi's Automation API lets you create declarative infrastructure defined by your best practices and expose it behind a REST, gRPC, or custom API.
```

Comment:
> lets you create declarative infrastructure

Just a general callout that I think it's important that we continue to drive home the message that this is the same declarative IaC that gives you all of the predictability, robustness, safety, and desired state management. We're just exposing those declarative building blocks over a new programmatic surface area. 

You can create custom infrastructure behind a REST API just using the raw AWS REST API or language specific SDKs. But the end result is not reliable and requires a massive amount of work to handle edge cases, error handling, group infrastructure into units and give it identity, etc. 

The doc does a pretty good job of this, but just thought it was worth reiterating.

Reply:
good point, added emphasis that it's the same building blocks with new programmatic surface area.